src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
+const ANIMAL_HEALTH = 100;
+
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = ANIMAL_HEALTH;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
+
+  static killedAnimal(victim) {
```

Comment:
in your case, the name is a noun, the function should answer the question “what to do?”
so you should rename to `kill` or `killAnimal`